packages/editor-ui/src/components/Workers/WorkerChartsAccordion.ee.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+	<WorkerAccordion icon="tasks" icon-color="black" :initial-expanded="false">
+		<template #title>
+			{{ $locale.baseText('workerList.item.chartsTitle') }}
+		</template>
+		<template #content>
+			<div :class="$style.charts">
+				<Chart
+					ref="chartRefJobs"
+					type="line"
+					:data="dataJobs"
+					:options="optionsJobs"
+					:class="$style.chart"
+				/>
+				<Chart
+					ref="chartRefCPU"
+					type="line"
+					:data="dataCPU"
+					:options="optionsCPU"
+					:class="$style.chart"
+				/>
+				<Chart
+					ref="chartRefMemory"
+					type="line"
+					:data="dataMemory"
+					:options="optionsMemory"
+					:class="$style.chart"
+				/>
+			</div>
+		</template>
+	</WorkerAccordion>
+</template>
+
+<script setup lang="ts">
+import WorkerAccordion from './WorkerAccordion.ee.vue';
+import { useOrchestrationStore } from '@/stores/orchestration.store';
+import { ref } from 'vue';
+import type { ChartData, ChartOptions } from 'chart.js';
+import {
+	Chart as ChartJS,
+	Title,
+	Tooltip,
+	Legend,
+	BarElement,
+	LineElement,
+	PointElement,
+	CategoryScale,
+	LinearScale,
+} from 'chart.js';
+import type { ChartComponentRef } from 'vue-chartjs';
+import { Chart } from 'vue-chartjs';
+import { averageWorkerLoadFromLoads, memAsGb } from './helpers';
+
+ChartJS.register(
+	CategoryScale,
+	LinearScale,
+	BarElement,
+	LineElement,
+	PointElement,
+	Title,
+	Tooltip,
+	Legend,
+);
```

Comment:
Can we move this inside a simple Vue plugin in `src/plugins/chartjs.ts`, and then `app.use()` in `main.ts` please? To avoid registering every time this component loads.